### DISCUSSION TITLE
Where can I get the xml scene of homogeneous_sss image sample

### DISCUSSION THREAD

**Discussion by boringfish** [mitsuba-renderer/mitsuba3#392] [Q&A] [answered]
Hi,
I try to render the same effect as the sample image from document: [https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_media.html](url)
and the sample image **bunny** is: 
![medium_homogeneous_sss](https://user-images.githubusercontent.com/32609638/202336822-817826c2-3681-499f-a891-989c6906007c.jpg)
But no matter how I adjust the parameters,  I can't achieve a similar effect. So I want to know is there a sample bunny xml scene I can get directly?

**Comment by njroussel** [comment, marked as answer]
Hi @boringfish 
All scenes used in the documentation are here: https://github.com/mitsuba-renderer/mitsuba-data/tree/master/docs/scenes
Specifically, I think you're looking for this one: https://github.com/mitsuba-renderer/mitsuba-data/blob/master/docs/scenes/medium_homogeneous_sss.xml

**Reply by boringfish** [reply]
Oh yes, I got it. Thank you very much
